unlock-protocol-com/package.json

```diff
@@ -4,6 +4,8 @@
   "private": true,
   "dependencies": {
     "@babel/core": "7.20.2",
+    "@ethersproject/abi": "^5.7.0",
+    "@ethersproject/providers": "^5.7.2",
```

Comment:
Pin dependency versions. 👍 

There are few other in the same file.

Reply:
resolved